### ISSUE TITLE
request for cityscapes related configs and pretrain weight

### ISSUE THREAD

**Issue by Cloveryww** [GuoleiSun/VSS-CFFM#4]
Hi,
Thanks for your great work firstly and could you please release configs and pretrain weights of cityscapes dataset?
I have seen part of files about cityscapes, but still lack of some configs file.

**Comment by GuoleiSun** [comment]
Thanks for your interest. Initially, we did not release the configs and pre-trained weights for cityscapes because video data of cityscapes is extremely large (>300 GB). Nevertheless, we will release the files for cityscapes later.

**Comment by Cloveryww** [comment]
Hi， 
I have some question about CFFM when running in cityscapes dataset.
1. When I want to train CFFM in cityscapes dataset, since training CFFM in cityscapes is a semi-supervised fasion as its sparse annotations, after I change the codes about dataset and loss compute(listed bellow), is any other important code needed adjustment?
loss compute:
https://github.com/GuoleiSun/VSS-CFFM/blob/d93f8a05283acaa0ecff2d0aeb11548fc8a25355/mmseg/models/decode_heads/decode_head.py#L752-L760
seg label in dataset:
https://github.com/GuoleiSun/VSS-CFFM/blob/d93f8a05283acaa0ecff2d0aeb11548fc8a25355/mmseg/datasets/custom.py#L1060-L1078

2. If the model run in cityscapes with a size of 512x1024, the params about core module of CFFM(such as bellow codes) is the same as  params in VSPW? If not, how to adjust them?  for example  "input_resolution"?
https://github.com/GuoleiSun/VSS-CFFM/blob/d93f8a05283acaa0ecff2d0aeb11548fc8a25355/mmseg/models/decode_heads/cffm_head.py#L80-L106

3. Does the current version of codes not support efficient inference on video data(reach the fps reported in paper), or  I did not find the corresponding codes?

Sincerely look forward to your reply, thank you!!!

**Comment by ydhongHIT** [comment]
> Thanks for your interest. Initially, we did not release the configs and pre-trained weights for cityscapes because video data of cityscapes is extremely large (>300 GB). Nevertheless, we will release the files for cityscapes later.

Hi, when are you going to provide the config files for cityscapes? And it is very kind to provide a brief instruction for dataset preparation. Thanks
